src/status_im/utils/datetime.cljs

```diff
@@ -25,12 +25,16 @@
   ([ms]
    (to-short-str ms #(unparse (formatters :hour-minute) %)))
   ([ms today-format-fn]
-   (let [date      (from-long ms)
-         local     (plus date time-zone-offset)
-         today     (t/today-at-midnight)
-         yesterday (plus today (days -1))]
+   (let [date        (from-long ms)
+         local       (plus date time-zone-offset)
+         today       (t/today-at-midnight)
+         yesterday   (plus today (days -1))
+         format-opts (clj->js {:day "numeric" 
+                               :month "short" 
+                               :hour "numeric" 
+                               :minute "numeric"})]
      (cond
-       (before? date yesterday) (unparse (formatter "dd MMM hh:mm") local)
+       (before? date yesterday) (.toLocaleString (js/Date. local) locale format-opts)
```

Comment:
If you think `.toLocaleString` is the right approach to localizing time stamps it might be worth moving it out into a function to be reused elsewhere?, ex: 
```clojure
(defn localize-timestamp
  ([timestamp] (localize-timestamp timestamp {}))
  ([timestamp opts] (.toLocaleString timestamp locale (clj->js opts)))
```